docs/sections/cn_workflows.md

```diff
@@ -528,3 +528,133 @@ This runs the step `one` locally in the host and step `two` through SLURM on any
 If `singularity` is used as the container engine, then by default the steps would run using MPI
 as SLURM jobs. This behaviour can be overriden by passing `mpi: false` in the configuration of the
 step for which MPI is not required.
+
+#### Host
+
+Popper executes the workflows by default using the `host` machine as the resource manager. So, when no resource manager is provided like the example below, the workflow runs on the local machine.
+
+```bash
+popper run -f sample.yml
+```
+
+The above assumes `docker` as the container engine and `host` as the resource manager to be
+used.
```

Comment:
all this section was removed in a recent commit, got back in this PR. Could you please remove it?

Reply:
Sorry about that! It's fixed now.